### PR TITLE
Create app component with Header and JobListings

### LINKED ISSUE #6: create app component
create an app component that renders all html

### PR DESCRIPTION
**Pull request by ksk5280**
closes #6

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -1,15 +1,39 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 
-var Root = React.createClass({
+var App = React.createClass({
```
Comment (natevenn) [inline lib/index.js]: Lets be consistent on using es6. Change to class Header extends React.Component

Comment (ksk5280) [inline lib/index.js]: Good catch, I'll go ahead and change it!

File: .jshintrc [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  "regexp": false,
+  "undef": true,
+  "sub": true,
+  "strict": false,
+  "white": false,
+  "eqnull": true,
+  "esnext": true,
+  "unused": true,
+  "mocha": true
+}
```
Comment (cheljoh) [inline .jshintrc]: Thanks for adding this!

**Comment by cheljoh** [comment]
Looks good, thanks for adding JSHint modules.

**Pull request merged**